### ISSUE TITLE
Extra Errors on Invocation of Abstract Constructor

### ISSUE THREAD

**Issue by aozgaa** [Microsoft/TypeScript#3711]
Consider the following code:

```ts
abstract class A {}
new A(1);
```

This should report one error, for invoking an abstract constructor. We currently report
`
error TS2346: Supplied parameters do not match any signature of call target.
`
`
error TS2511: Cannot create an instance of the abstract class 'A'.
`

so we should eliminate the first error.
